docs/docs/api/http.md

```diff
@@ -643,3 +643,24 @@ The response comes in two parts:
   - `total`
 
     The total number of elements across all pages.
+
+### Metadata
+
+Please refer to our [metadata](glossary.md#metadata) definition for more
+information.
+
+### Setting metadata
+
+Airy provides a mechanism to add metadata to a conversation.
+
+`POST /metadata.set`
+
+```json
+{
+  "conversation_id": "conversation-id",
+  "key": "source.contact.first_name",
```

Comment:
```suggestion
  "key": "ad.id",
```

Proposing a different example so we don't confuse readers with actual keys we use internally